src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -749,3 +749,10 @@ COOP_PINVOKE_HELPER(void, RhGetCurrentThreadStackBounds, (PTR_VOID * ppStackLow,
 {
     ThreadStore::GetCurrentThread()->GetStackBounds(ppStackLow, ppStackHigh);
 }
+
+ThreadExitCallback g_threadExitCallback;
```

Comment:
What would be the benefit from it?

Adding it on Windows requires extra plumbing in the ProjectN build and so it should be better done on the TFS side.